### PR TITLE
Dir packet implementation and removed useless "mv" packet

### PR DESCRIPTION
**Pull request by Adanlink**
0Lucifer0 helped me.

### CODE REVIEW

File: src/NosCore.Packets/ServerPackets/DirPacket.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using NosCore.Core.Serializing;
+using NosCore.Shared.Enumerations;
+using System.ComponentModel.DataAnnotations;
+
+namespace NosCore.Packets.ClientPackets
+{
+    [PacketHeader("dir")]
+    public class DirPacket : PacketDefinition
```
Comment (Elendan) [inline src/NosCore.Packets/ServerPackets/DirPacket.cs]: Why is there a second dir packet in the Client packets?

**Comment by Elendan** [comment]
Why has the code been completely re indented ? can't review that

**Comment by Adanlink** [comment]
Srry I needed to upload the files via webpage. And yeah, I messed up with the two packets. Dir_Packet.cs needs to go to the Server Packets and DirPacket.cs to the client one. When I arrive home I will fix it.

**Comment by 0Lucifer0** [comment]
This is unreviewable

**Pull request closed without merging**